test/unit/modifySongDataTest.js

```diff
@@ -2,10 +2,15 @@ const test = require('tape');
 const modifySongData = require('../../src/modifySongData');
 const songData = require('../../metadata/jazzy_beats.json');
 
+function songDataWithCustomOpts(energyCustomization) {
+  return Object.assign({}, songData, { energyCustomization });
```

Comment:
Nonblocking, just curious: Do we not support object-spread in this repo yet?  Or is there a reason you prefer `Object.assign`?
```js
return {...songData, energyCustomization};
```

Reply:
That is correct. It seemed to work in the test (at least when I ran via `node`), but `...` did not work in the main code base, so I decided to be consistent about using `assign`